fondant/schemas/component_spec.json

```diff
@@ -2,7 +2,14 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Example Component Schema",
   "type": "object",
-  "required": ["name", "description", "image", "input_subsets", "output_subsets", "args"],
+  "required": [
+    "name",
+    "description",
+    "image",
```

Comment:
I was wondering whether the "image" is relevant for the Fondant spec, or whether this is only relevant for the Kubeflow spec. For the moment this field is not used

Reply:
It's necessary to build the kubeflow spec. I think it's logical that fondant components need to define the image they'll be executed with.